app/indexers/concerns/iiif_print/file_set_indexer.rb

```diff
@@ -19,10 +19,7 @@ def generate_solr_document
         # only UV viewable images should have is_page_of, it is only used for iiif search
         solr_doc['is_page_of_ssim'] = iiif_print_lineage_service.ancestor_ids_for(object) if object.mime_type&.match(/image/)
         # index for full text search
-        text = IiifPrint::Data::WorkDerivatives.data(from: object, of_type: 'txt')
-        text = text.tr("\n", ' ').squeeze(' ')
-        solr_doc['all_text_timv'] = text
-        solr_doc['all_text_tsimv'] = text
+        solr_doc['all_text_timv'] = solr_doc['all_text_tsimv'] = all_text
```

Comment:
```suggestion
        solr_doc['all_text_timv'] = all_text
        solr_doc['all_text_tsimv'] = all_text
```

Reply:
is this what you meant? if not, how does the change you wrote work?

Reply:
I did mean to do that because I wanted to shorten it to just one line since they both need to be set to the same value.  This will chain the assignment of those index fields to the same thing.

From what I read, the difference between assigning them in multiple lines is that when chaining the assignments together, they will both point to the same object id, which I believe will be fine in this case and shouldn't cause any problems.

ex.
```rb
a = b = 'foo'
a #=> 'foo'
a.object_id #=> 442000
b #=> 'foo'
b.object_id #=> 442000

c = 'bar'
d = 'bar'
c #=> 'bar'
c.object_id #=> 442020
d #=> 'bar'
d.object_id #=> 442040
```

Reply:
I would probably do it like this if you want it in one line: 
```
solr_doc['all_text_timv'], solr_doc['all_text_tsimv'] = all_text, all_text
```

Reply:
Sure i can change it to whatever and it really doesn't have to be on one line, i just didn't want all_text to be evaluated again but in the grand scheme of things it's probably not going to be too bad.

I'm just curious what's wrong with the way I did it?

Reply:
I hadn't seen the chain assignment before; that's pretty dope! I wouldn't have minded you keeping it that way since you explained it.

I'll approve with the unchained version, but if you wanna chain them again I'm fine with that too.